config.yaml

```diff
@@ -1,14 +1,4 @@
 # Copyright 2021 Unicorn
```

Comment:
maybe this file can be removed too, if there are no config options? Not sure about it but you can give it a try.

Reply:
There could be others here, because I'm already starting to work on this (there will be a new PR).